FitGoal/FitGoal/GoalTrakerCell.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  GoalTrakerCell.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 8/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class GoalTrakerCell: UICollectionViewCell {
+    static var identifier = "GoalTraker"
+    
+    lazy var topView: UIView = {
+        let gradientView = GradientView(frame: CGRect(x: 0, y: 0, width: 800, height: 812))
+        gradientView.layer.cornerRadius = 150
+        gradientView.layer.maskedCorners = [.layerMinXMaxYCorner]
+        gradientView.colors = [#colorLiteral(red: 0.2816967666, green: 0.8183022738, blue: 0.9222241044, alpha: 1), #colorLiteral(red: 0.5647058824, green: 0.07450980392, blue: 0.9568627451, alpha: 1)]
+        let rotation = CGAffineTransform(rotationAngle: -26 / 180 * CGFloat.pi)
+        gradientView.transform = rotation.translatedBy(x: 15, y: -650)
+        return gradientView
+    }()
+    
+    lazy var cellTitle: UILabel = {
+        let label = UILabel()
+        label.attributedText = "MY GOAL".formattedText(
+            font: "Oswald-Medium",
+            size: 34,
+            color: .white,
+            kern: -0.14
+        )
+        return label
+    }()
+
+    lazy var monthLabel: UILabel = {
+        let label = UILabel()
+        label.attributedText = "February".formattedText (
+            font: "Roboto-Regular",
+            size: 15,
+            color: .white,
+            kern: 0.18
+        )
+        return label
+    }()
+    
+    var imageAvatar: UIImageView = {
+        let avatar = UIImageView(image: UIImage(imageLiteralResourceName: "Avatar"))
+        avatar.frame = CGRect(x: 0, y: 0, width: 42, height: 42)
+        avatar.contentMode = .scaleAspectFill
+        return avatar
+    }()
+    var addButton: UIButton = {
```

Comment:
SPACES!!!
Revise the spaces in this file, there are some of them missing and too many in other cases.